examples/p4est_2d_dgsem/elixir_navierstokes_NACA0012airfoil_mach08.jl

```diff
@@ -130,12 +130,26 @@ lift_coefficient = AnalysisSurfaceIntegral(semi, force_boundary_names,
                                                                    u_inf(equations),
                                                                    l_inf()))
 
+drag_coefficient_force = AnalysisSurfaceIntegral(semi, force_boundary_names,
```

Comment:
This is a better distinction because even the coefficient of pressure is a force.
```suggestion
drag_coefficient_shear = AnalysisSurfaceIntegral(semi, force_boundary_names,
```